src/core/api/index.js

```diff
@@ -162,111 +161,22 @@ export function callApi({
     .then((response) => (schema ? normalize(response, schema) : response));
 }
 
-type FetchAddonParams = {|
-  api: ApiStateType,
-  slug: string,
-|};
-
-export function fetchAddon({ api, slug }: FetchAddonParams) {
-  return callApi({
-    endpoint: `addons/addon/${slug}`,
-    schema: addon,
-    auth: true,
-    state: api,
-  });
-}
-
-type LoginParams = {|
-  api: ApiStateType,
-  code: string,
-  state: string,
-|};
-
-export function login({ api, code, state }: LoginParams) {
-  const params = { config: undefined };
-  const configName = config.get('fxaConfig');
-  if (configName) {
-    params.config = configName;
-  }
-  return callApi({
-    endpoint: 'accounts/login',
-    method: 'POST',
-    body: { code, state },
-    params,
-    state: api,
-    credentials: true,
-  });
-}
-
-export function startLoginUrl(
-  { location }: {| location: ReactRouterLocation |},
-) {
-  const configName = config.get('fxaConfig');
-  const params = {
-    config: undefined,
-    to: url.format({ ...location }),
-  };
-  if (configName) {
-    params.config = configName;
-  }
-  const query = makeQueryString(params);
-  return `${API_BASE}/accounts/login/start/${query}`;
-}
-
-type FeaturedParams = {|
-  api: ApiStateType,
-  filters: Object,
-  page: number,
-|};
-
-export function featured({ api, filters, page }: FeaturedParams) {
-  return callApi({
-    endpoint: 'addons/featured',
-    params: {
-      app: api.clientApp,
-      ...convertFiltersToQueryParams(filters),
-      page,
-    },
-    schema: { results: [addon] },
-    state: api,
-  });
-}
-
-export function categories({ api }: {| api: ApiStateType |}) {
-  return callApi({
-    endpoint: 'addons/categories',
-    schema: { results: [category] },
-    state: api,
-  });
-}
-
-export function logOutFromServer({ api }: {| api: ApiStateType |}) {
-  return callApi({
-    auth: true,
-    credentials: true,
-    endpoint: 'accounts/session',
-    method: 'DELETE',
-    state: api,
-  });
-}
-
-type AutocompleteParams = {|
-  api: ApiStateType,
-  filters: {|
-    query: string,
-    addonType?: string,
-  |},
-|};
-
-export function autocomplete({ api, filters }: AutocompleteParams) {
-  return callApi({
-    endpoint: 'addons/autocomplete',
-    params: {
-      app: api.clientApp,
-      ...convertFiltersToQueryParams(filters),
-    },
-    state: api,
-  });
-}
-
-export const search = searchApi;
+// Schemas
+// TODO: Remove this when
+// https://github.com/mozilla/addons-frontend/issues/2917 is fixed.
+export const addon = addonApi.addonSchema;
+export const category = categoriesApi.categorySchema;
+
+// Add-on APIs
+export const fetchAddon = addonApi.fetchAddon;
```

Comment:
I don't see the benefit in aliasing all of these submodules. It just creates confusion and requires a bunch of boilerplate code. Why can't we just update all of the import statements? If the test failures are confusing, you could could push a commit with the aliases removed and I'll take a look.